xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -473,6 +514,90 @@ mod tests {
 		assert_eq!(inverted, Err(()));
 	}
 
+	#[test]
+	fn global_consensus_converts_for_works() {
+		parameter_types! {
+			pub UniversalLocation: InteriorMultiLocation = X2(GlobalConsensus(ByGenesis([9; 32])), Parachain(1234));
+		}
+
+		let test_data = vec![
+			(MultiLocation::parent(), false),
+			(MultiLocation::new(0, Here), false),
+			(MultiLocation::new(0, X1(GlobalConsensus(ByGenesis([9; 32])))), false),
```

Comment:
You could move `ByGenesis([9; 32])` and `ByGenesis([0; 32])` to constants. `9` and `0` are pretty similar :sweat_smile: